### PR TITLE
added new columns to classroom, workgroup, worksheet tables

### PR DESCRIPTION
**Pull request by AnnK89**
@JulienErgan Could you take a look and let me know if I missed out anything?

### CODE REVIEW

**Review by JulienErgan — APPROVED** [review]

File: app/models/work_group.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@
 #
 #  id              :bigint           not null, primary key
 #  name            :string
+#  time_limit      :integer
```
Comment (Naokimi) [inline app/models/work_group.rb:7]: would be good to add a comment of what it is - minutes? seconds?

Comment (Naokimi) [reply app/models/work_group.rb:7]: also needs to add rotation time

File: db/migrate/20200627060227_add_name_to_classrooms.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddNameToClassrooms < ActiveRecord::Migration[6.0]
```
Comment (Naokimi) [inline db/migrate/20200627060227_add_name_to_classrooms.rb:1]: please merge migrations that change things in the same model
i.e.: we don't need to separate migrations to add a column, but create a migration with a different name and add the two change actions in there

**Review by Naokimi — CHANGES_REQUESTED** [review]

File: app/models/work_group.rb (hunk 1 above)
Comment (AnnK89) [reply app/models/work_group.rb:7]: okay understood, will get it fixed 👍🏻

**Pull request closed without merging**